### PR TITLE
Relatedproducts

### PR DESCRIPTION
**Pull request by acerslee**
Requesting review. Refactored code to tailor to API file functions.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -10,18 +10,24 @@ class App extends React.Component{
...
     return(
       <div>
         <Overview />
-        <RelatedList />
-        <YourOutfitList />
-        <Questions product_id={this.state.product_id}/>
-        <Reviews product_id={this.state.product_id}/>
+        <RelatedList product_id = {this.state.product_id} renderNewProductId = {this.renderNewProductId.bind(this)}/>
+        <YourOutfitList product_id = {this.state.product_id}/>
+        <Questions />
```
Comment (bmh0013) [inline client/components/App.jsx:29]: Looks like not everything was pulled down from production, both Questions and Reviews should have items being passed down as props

File: client/components/Overview/serverRequests.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import API_KEY from '../../../token.js';
```
Comment (bmh0013) [inline client/components/Overview/serverRequests.js:1]: Shouldnt the code from this file be in API.js and therefore this can be deleted?

**Review by bmh0013 — COMMENTED** [review]
Everything else looks good

File: client/components/App.jsx (hunk 1 above)
Comment (acerslee) [reply client/components/App.jsx:29]: @bmh0013 Yeah I forgot to uncomment the other components before I did the push.

**Review by koalarick — APPROVED** [review]
I'm good with changes.

**Pull request merged**